### PR TITLE
Update Satellite installation method according satellite offical installation guide

### PR DESCRIPTION
**Pull request by yanpliu**
Satellite6.12 installation failed for error
```
Command: subscription-manager register --org Sat6-CI --activationkey 'satellite-6.12-qa-rhel8'
Retcode: 70
Output:
Couldn't find activation key 'satellite-6.12-qa-rhel8' (HTTP error code 404: Not Found)
```
Update Satellite installation guide according satellite offical installation guide
http://ohsnap.sat.engineering.redhat.com/streams/6.12/releases/6.12.0/snaps/4.0/installation

Run: PASS
```
(virtwho_ci_vv) [yanpliu@yanpliu virtwho-ci]$ python3 ${WORKSPACE}/${VIRTWHO_CI}/utils/installer.py install-satellite     --host=XXXXX --username=root     --password=XXX --sat-release=satellite6.12     --sat-repo=dogfood --sat-os=RHEL-8.6.0-20220510.3
/home/yanpliu/data/virtualenv_38/virtwho_ci_vv/lib/python3.8/site-packages/paramiko/transport.py:216: CryptographyDeprecationWarning: Blowfish has been deprecated
  "class": algorithms.Blowfish,
2022-08-02 13:55:49 [INFO] Succeeded to connect host dell-per740-68-vm-10.lab.eng.pek2.redhat.com by ssh
2022-08-02 13:55:49 [INFO] rhsm.conf is backup already(dell-per740-68-vm-10.lab.eng.pek2.redhat.com)
2022-08-02 13:55:56 [INFO] Finished to init system dell-per740-68-vm-10.lab.eng.pek2.redhat.com
2022-08-02 13:56:03 [INFO] Succeeded to unregister and clean system(dell-per740-68-vm-10.lab.eng.pek2.redhat.com)
2022-08-02 13:56:57 [INFO] Succeeded to register (dell-per740-68-vm-10.lab.eng.pek2.redhat.com) to Stage Candlepin
2022-08-02 13:58:11 [INFO] Succeeded to attach Employee SKU(dell-per740-68-vm-10.lab.eng.pek2.redhat.com)
2022-08-02 13:59:53 [INFO] Succeeded to enable satellite repos
2022-08-02 13:59:55 [INFO] Succeeded to update ca cert
2022-08-02 13:59:57 [INFO] Succeeded to get satellite repo_file
2022-08-02 14:09:46 [INFO] Start to install satellite package(dell-per740-68-vm-10.lab.eng.pek2.redhat.com)
2022-08-02 14:11:29 [INFO] Succeeded to install satellite package(dell-per740-68-vm-10.lab.eng.pek2.redhat.com)
2022-08-02 14:13:32 [INFO] Succeeded to run satellite-installer --scenario satellite(dell-per740-68-vm-10.lab.eng.pek2.redhat.com)
2022-08-02 14:14:09 [INFO] Succeeded to upload manifest to satellite(dell-per740-68-vm-10.lab.eng.pek2.redhat.com)
2022-08-02 14:14:25 [INFO] Succeeded to refresh satellite manifest(dell-per740-68-vm-10.lab.eng.pek2.redhat.com)
2022-08-02 14:14:25 [INFO] Succeeded to deploy satellite(dell-per740-68-vm-10.lab.eng.pek2.redhat.com)
2022-08-02 14:14:36 [INFO] Succeeded to copy sshkey from dell-per740-68-vm-10.lab.eng.pek2.redhat.com to 10.73.131.217

```

### CODE REVIEW

File: virt_who/provision.py [hunk 1]
```diff
@@ -1402,6 +1402,36 @@ def satellite_qa_dogfood_enable(self, ssh_sat, sat_ver, rhel_ver, repo_type="sat
...
+            raise FailException("Failed to update ca cert)".format(ssh_sat))
+        logger.info("Succeeded to update ca cert".format(ssh_sat))
+        cmd = "curl -o /etc/yum.repos.d/satellite.repo http://ohsnap.sat.engineering.redhat.com/api/releases/{0}.0/el{1}/satellite/repo_file".format(
+            sat_ver, rhel_ver)
+        status, output = self.run_loop(cmd, ssh_sat, desc="get satellite repo_file")
+        if status != "Yes":
+            raise FailException("Failed to get satellite repo_file)".format(ssh_sat))
+        logger.info("Succeeded to get satellite repo_file".format(ssh_sat))
+        if rhel_ver == "8":
+            cmd = "dnf -y module enable satellite:el8"
```
Comment (hkx303) [inline virt_who/provision.py:1432]: what if  `rhel_ver == "7" ` ?

**Review by hkx303 — COMMENTED** [review]
ACK, pending a small question

File: virt_who/provision.py (hunk 1 above)
Comment (hkx303) [reply virt_who/provision.py:1432]: my bad, we shouldn't do anything if `rhel_ver == "7"`

**Review by hkx303 — APPROVED** [review]
Good job

File: virt_who/provision.py (hunk 1 above)
Comment (yanpliu) [reply virt_who/provision.py:1432]: @hkx303 this step is a workaround for rhel_ver  is 8 from satellite installation guide, for rhel-ver is 7, it is no need to do this step.
```
3. Run the install steps

dnf -y module enable satellite:el8
dnf -y install satellite
satellite-installer --scenario satellite
```
Please reference rhel-ver is 8 installation guide 
http://ohsnap.sat.engineering.redhat.com/streams/6.12/releases/6.12.0/snaps/4.0/installation

Please reference rhel-ver is 7 installation guide 
http://ohsnap.sat.engineering.redhat.com/streams/6.11/releases/6.11.1.1/snaps/2.0/installation

**Comment by yanpliu** [comment]
Thank you very much kunxin

**Pull request merged**